<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WriterWindow</class>
 <widget class="QWidget" name="WriterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WriterWindow</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>411</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="descriptionLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Act:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="actLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>239</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="acceptButton">
      <property name="text">
       <string>Accept</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>140</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="developButton">
      <property name="text">
       <string>Develop</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="savePlotButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>250</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Plot</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>290</y>
     <width>281</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>WriterWindow</receiver>
   <slot>handleAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptButton</sender>
   <signal>clicked()</signal>
   <receiver>WriterWindow</receiver>
   <slot>handleAccept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>WriterWindow</receiver>
   <slot>handleSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>developButton</sender>
   <signal>clicked()</signal>
   <receiver>WriterWindow</receiver>
   <slot>handleDevelop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>savePlotButton</sender>
   <signal>clicked()</signal>
   <receiver>WriterWindow</receiver>
   <slot>handleSavePlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>handleAdd()</slot>
  <slot>handleAccept()</slot>
  <slot>handleDevelop()</slot>
  <slot>handleSave()</slot>
  <slot>handleSavePlot()</slot>
 </slots>
</ui>
